src/styles/main.scss

```diff
@@ -0,0 +1,82 @@
+$day_background-color: #eee;
```

Comment:
This should be in separate file for example in `_variables.scss`. Also I don't think its necessary to put those values in variables as it is used only in one place.